include/ck_tile/ops/gemm/kernel/gemm_tile_partitioner.hpp

```diff
@@ -1,73 +1,130 @@
 // SPDX-License-Identifier: MIT
-// Copyright (c) 2018-2024, Advanced Micro Devices, Inc. All rights reserved.
+// Copyright (c) 2018-2025, Advanced Micro Devices, Inc. All rights reserved.
 
 #pragma once
 
 #include "ck_tile/core.hpp"
 
 namespace ck_tile {
-template <typename BlockGemmShape_>
-struct GemmTilePartitioner
+
+/// @brief An internal ck-tile class dedicated to 2D tiles.
```

Comment:
Please elaborate on the purpose of this class. From the above information I still do not know what this class is responsible for and don't know how to correlate those 2D tiles with that.